src/deadline/client/api/_queue_parameters.py

```diff
@@ -22,9 +22,13 @@ def get_queue_parameter_definitions(
     *, farmId: str, queueId: str, config=None
 ) -> list[JobParameter]:
     """
-    This gets all the queue parameters definitions from the specified Queue. It does so
-    by getting all the full templates for queue environments, and then combining
+    This gets all the queue parameters definitions from the specified [Deadline Cloud queue].
```

Comment:
Pretty sure this is just a personal preference. But I think of it like `gets for` vs. `retrieves from`

```suggestion
    This gets all the queue parameter definitions for the specified [Deadline Cloud queue].
```